ports/angle/portfile.cmake

```
@@ -57,6 +57,20 @@ configure_file("${CMAKE_CURRENT_LIST_DIR}/angle_commit.h.in" "${SOURCE_PATH}/ang
 configure_file("${CMAKE_CURRENT_LIST_DIR}/angle_commit.h.in" "${SOURCE_PATH}/src/common/angle_commit.h" @ONLY)
 file(COPY "${CMAKE_CURRENT_LIST_DIR}/unofficial-angle-config.cmake" DESTINATION "${SOURCE_PATH}")
 
+if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
+    file(
+        APPEND
+        "${SOURCE_PATH}/unofficial-angle-config.cmake"
+            "\n\
+foreach(angle_target EGL GLESv2 ANGLE)\n\
+    if(TARGET \"unofficial::angle::lib\${angle_target}\")\n\
+        target_compile_definitions(\"unofficial::angle::lib\${angle_target}\" INTERFACE KHRONOS_STATIC)\n\
+    endif()\n\
```

```suggestion
    endif()
```
We don't need newlines symbol `\n\`. :)

---

Sorry, I didn't understand your suggestion. This is the result before your suggestion:

```
foreach(angle_target EGL GLESv2 ANGLE)
    if(TARGET "unofficial::angle::lib${angle_target}")
        target_compile_definitions("unofficial::angle::lib${angle_target}" INTERFACE KHRONOS_STATIC)
    endif()
endforeach()
unset(angle_target)
```

This is the result after your suggestion:

```
foreach(angle_target EGL GLESv2 ANGLE)
    if(TARGET "unofficial::angle::lib${angle_target}")
        target_compile_definitions("unofficial::angle::lib${angle_target}" INTERFACE KHRONOS_STATIC)
    endif()endforeach()
unset(angle_target)
```

---

Okay, I understand now. I can use a multiline string. I will change this now.